### PR TITLE
add code to readme

### LINKED ISSUE #739: Add code to Readme
### Description

It was moved from readme to quick start. We need some good code examples in readme. Otherwise, it does not look like a developer tool 🙂

### PR DESCRIPTION
**Pull request by dmpetrov**
Closes #739

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -42,6 +42,88 @@ Getting Started
...
+
+Example: LLM based text-file evaluation
+---------------------------------------
+
+In this example, we evaluate chatbot conversations stored in text files
+using LLM based evaluation.
+
+.. code:: shell
+
+    $ pip install mistralai (Requires version >=1.0.0)
```
Comment (dreadatour) [inline README.rst:82]:
```suggestion
    $ pip install mistralai # Requires version >=1.0.0
```

**Review by dreadatour — APPROVED** [review]
👍

File: README.rst [hunk 2]
```diff
@@ -42,6 +42,88 @@ Getting Started
 Visit `Quick Start <https://docs.datachain.ai/quick-start>`_ and `Docs <https://docs.datachain.ai/>`_
 to get started with `DataChain` and learn more.
 
+.. code:: bash
+
+        pip install datachain
+
+
+Example: download subset of files based on metadata
```
Comment (skshetry) [inline README.rst:50]: Should we first start with "querying" feature?

Comment (dmpetrov) [reply README.rst:50]: Could you please explain more?

**Pull request merged**